spec/java.common.lsl

```diff
@@ -108,19 +108,40 @@ define action CALL_METHOD (
     ): any;
 
 
+/*
+Usage example:
+action TRY_CATCH(
+    _try_proc(),
+    [
+        ["java.lang.Error",     _catch_proc_a()],
+        ["java.util.Exception", _catch_proc_b()],
+    ]
+);
+*/
+define action TRY_CATCH (
+        tryBlock: any,                // subroutine call!
+        catchTable: array<array<any>> // pairs of {name: str, handler: call}
+    ): void;
+
+
+// WARNING: applicable only within the exception handler ("catch") subroutine
+define action CATCH_GET_EXCEPTION_REF (
```

Comment:
I didn't understand how we will use this action, can you explain me please ? Or write a little example ?